runtime/dali/src/weights/mosaic.rs

```diff
@@ -0,0 +1,110 @@
+
+//! Autogenerated weights for `mosaic`
+//!
+//! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 4.0.0-dev
```

Comment:
how it was generated? machine generated that must be same spec as our runtime running i guess

Reply:
this will be overwritten by the benchmark ran in the CI I believe, it's more like a placeholder to make the CI happy

Reply:
i see. so can put here any weight even like 100_000 for each value?  so that we have not need to run it locally